src/action.ts

```diff
@@ -41,7 +39,7 @@ export const runMeticulousTestsAction = async (): Promise<void> => {
   // Init Sentry without sampling traces on the action run.
   // Children processes, (test run executions) will use
   // the global sample rate.
-  const sentryHub = await initSentry(1.0);
+  const sentryHub = await initSentry("report-diffs-action-v1", 1.0);
```

Comment:
This is no longer the SDK version.

When we merge it into the Meticulous repo it'll all switch round a bit though -- we'll be using the git commit hash instead. I can then export that from the bundle (commitSha), so can specify the specific version here.